script/vercel-setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Fail on all sorts of errors.
+# https://stackoverflow.com/a/2871034/2116927
+set -euxo pipefail
+
+# In Vercel -> Project Settings -> Build & Development Settings:
+# Build command   : yarn vercel:build
+# Output directory: vercel-static
+# Install command : yarn vercel:install
```

Comment:
@tangjeff0 these options should work, and not need the secrets from https://github.com/athensresearch/athens/pull/1894#discussion_r765677397.